webofneeds/won-owner-webapp/src/main/java/won/owner/web/websocket/WonWebSocketHandler.java

```diff
@@ -242,30 +245,37 @@ private void notifyPerEmail(final User user, final URI needUri, final WonMessage
         case OPEN:
           if (userNeed.isConversations()) {
             emailSender.sendConversationNotificationHtmlMessage(
-              user.getEmail(), needUri.toString(), wonMessage.getSenderNeedURI().toString(), textMsg);
+              user.getEmail(), needUri.toString(), wonMessage.getSenderNeedURI().toString(), wonMessage.getReceiverURI().toString(), textMsg);
           }
           return;
         case CONNECTION_MESSAGE:
           if (userNeed.isConversations()) {
             emailSender.sendConversationNotificationHtmlMessage(
-              user.getEmail(), needUri.toString(), wonMessage.getSenderNeedURI().toString(), textMsg);
+              user.getEmail(), needUri.toString(), wonMessage.getSenderNeedURI().toString(), wonMessage.getReceiverURI().toString(), textMsg);
           }
           return;
         case CONNECT:
           if (userNeed.isRequests()) {
             emailSender.sendConnectNotificationHtmlMessage(
-              user.getEmail(), needUri.toString(), wonMessage.getSenderNeedURI().toString(), textMsg);
+              user.getEmail(), needUri.toString(), wonMessage.getSenderNeedURI().toString(), wonMessage.getReceiverURI().toString(), textMsg);
           }
           return;
         case HINT_MESSAGE:
```

Comment:
as i said in the comment earlier im not sure if we should keep this, however i think it does not work anyway.
I tested it again and i do not receive any hint messages when im logged out